crates/holochain_zome_types/src/schedule.rs

```diff
@@ -1,8 +1,28 @@
+use crate::FunctionName;
+use crate::ZomeName;
 use std::time::Duration;
 
+/// Tick the scheduler every this many millis.
+pub const SCHEDULER_INTERVAL_MILLIS: u64 = 10000;
+
+/// Expire persisted schedules after this long.
+pub const PERSISTED_TIMEOUT_MILLIS: i64 = 20000;
```

Comment:
consider using const Durations to further alleviate unit confusion.

```suggestion
/// Tick the scheduler every this many millis.
pub const SCHEDULER_INTERVAL: Duration = Duration::from_millis(10000);

/// Expire persisted schedules after this long.
pub const PERSISTED_TIMEOUT: Duration = Duration::from_millis(20000);
```